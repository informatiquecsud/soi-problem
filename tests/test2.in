10
Q
C 8
C 7
C 11
Q
C 2
C 4
C 3
Q
C 3