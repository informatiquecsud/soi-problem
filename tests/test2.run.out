0
1
3
**********************************************************************
File "/workspace/soi-problem/src/task.py", line 35, in __main__.find_last_height_geq
Failed example:
    last_geq
Expected nothing
Got:
    [0, 0, 0, 0, 0, 0, 0]
**********************************************************************
File "/workspace/soi-problem/src/task.py", line 36, in __main__.find_last_height_geq
Failed example:
    find_last_height_geq(heights, 1)
Exception raised:
    Traceback (most recent call last):
      File "/home/gitpod/.pyenv/versions/3.11.9/lib/python3.11/doctest.py", line 1355, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest __main__.find_last_height_geq[4]>", line 1, in <module>
        find_last_height_geq(heights, 1)
    TypeError: find_last_height_geq() missing 1 required positional argument: 'i'
**********************************************************************
File "/workspace/soi-problem/src/task.py", line 38, in __main__.find_last_height_geq
Failed example:
    find_last_height_geq(heights, 3)
Exception raised:
    Traceback (most recent call last):
      File "/home/gitpod/.pyenv/versions/3.11.9/lib/python3.11/doctest.py", line 1355, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest __main__.find_last_height_geq[5]>", line 1, in <module>
        find_last_height_geq(heights, 3)
    TypeError: find_last_height_geq() missing 1 required positional argument: 'i'
**********************************************************************
File "/workspace/soi-problem/src/task.py", line 40, in __main__.find_last_height_geq
Failed example:
    find_last_height_geq(heights, 4)
Exception raised:
    Traceback (most recent call last):
      File "/home/gitpod/.pyenv/versions/3.11.9/lib/python3.11/doctest.py", line 1355, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest __main__.find_last_height_geq[6]>", line 1, in <module>
        find_last_height_geq(heights, 4)
    TypeError: find_last_height_geq() missing 1 required positional argument: 'i'
**********************************************************************
File "/workspace/soi-problem/src/task.py", line 42, in __main__.find_last_height_geq
Failed example:
    find_last_height_geq(heights, 5)
Exception raised:
    Traceback (most recent call last):
      File "/home/gitpod/.pyenv/versions/3.11.9/lib/python3.11/doctest.py", line 1355, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest __main__.find_last_height_geq[7]>", line 1, in <module>
        find_last_height_geq(heights, 5)
    TypeError: find_last_height_geq() missing 1 required positional argument: 'i'
**********************************************************************
File "/workspace/soi-problem/src/task.py", line 44, in __main__.find_last_height_geq
Failed example:
    find_last_height_geq(heights, 6)
Exception raised:
    Traceback (most recent call last):
      File "/home/gitpod/.pyenv/versions/3.11.9/lib/python3.11/doctest.py", line 1355, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest __main__.find_last_height_geq[8]>", line 1, in <module>
        find_last_height_geq(heights, 6)
    TypeError: find_last_height_geq() missing 1 required positional argument: 'i'
**********************************************************************
1 items had failures:
   6 of   9 in __main__.find_last_height_geq
***Test Failed*** 6 failures.
